src/conic.jl

```diff
@@ -879,7 +912,7 @@ function MathProgBase.optimize!(m::PajaritoConicModel)
 
     # Release the inf_conic_model if applicable
     if applicable(MathProgBase.freemodel!,ini_conic_model)
-        MathProgBase.freemodel!(ini_conic_model)
+        !contains(string(typeof(m.cont_solver)), "MosekSolver") && MathProgBase.freemodel!(ini_conic_model)
```

Comment:
This is already fixed in mosek (https://github.com/JuliaOpt/Mosek.jl/pull/71), we don't need a special case.